docs/src/documentation/02-foundation/05-design-tokens/02-installation.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Installation
```

Comment:
Could we have some description here? Maybe

```suggestion
title: Installation
description: How to install our design tokens package
```

Reply:
interesting, I accepted ur suggestion